lms/static/sass/base/_grid-settings.scss

```diff
@@ -7,7 +7,7 @@ $border-box-sizing: false;
 
 
 /* Breakpoints */
-$mobile: new-breakpoint(max-width 320px 4);
-$tablet: new-breakpoint(min-width 321px max-width 768px, 8);
+$mobile: new-breakpoint(max-width 540px 4);
```

Comment:
I'd also like to know the reason for this change. From the diff, I'm assuming that there's a need for a  new 540px - 768px range to support some layout elements. If this is the case, I'd recommend not adjusting the base ``$mobile`` breakpoint, but adding another breakpoint  in between ``$mobile`` and ``$tablet``. Perhaps something like this:

```scss
$mobile: new-breakpoint(max-width 320px 4);
$small-tablet: new-breakpoint(min-width 321px max-width 540px, 4);
$tablet: new-breakpoint(min-width 541px max-width 768px, 8);
```

Also, a few thoughts about this new addition:

1. You are changing the column count from 8 (what the previous ``$tablet`` breakpoint defined) to 4 for the range of 321px - 540px. This amy affect other UI elements that are assuming the 8 column layout. Please proof these (I believe @AlasdairSwan added these breakpoints and only the header/footer UI elements use them now).

2. This naming convention is getting a bit hairy and not handling cases like this well. I'd recommend that we move to changing these names/sizes to our emerging standards (https://github.com/edx/ux-pattern-library/wiki/Styleguide:-Sass-&-CSS#responsive-breakpoints) so that they are device agnostic.

Hope that helps. Please ping me if you have any questions. I'll be reviewing this PR in more detail shortly.